src/Node/ParityNode.ts

```diff
@@ -32,14 +33,26 @@ export default class ParityNode extends Node {
       '--volume', `${this.chainDir}:/home/parity/.local/share/io.parity.ethereum/`,
     ]);
 
+    // Running the parity process inside the container as the same user id that is executing this
+    // script.
+    // This is required, because otherwise the parity process will not be able to write to the
+    // mounted directory. The parity process inside the container is not run as root (as usual),
+    // but instead runs with uid/guid 1000/1000 by default. This option overrides that default
+    // behavior so that the parity process can write to its mounted chain directory in all
+    // environments. This was introduced after failing tests on Travis CI.
+    const userInfo = os.userInfo();
+    args = args.concat([
+      '--user', `${userInfo.uid}:${userInfo.gid}`,
+    ]);
+
     if (this.password !== '') {
       args = args.concat([
         '--volume', `${this.password}:/home/parity/password.txt`,
       ]);
     }
 
     args = args.concat([
-      'parity/parity:v2.3.4',
+      'parity/parity:v2.3.3',
```

Comment:
In the PR description, you mentioned bumping parity version to `2.4.5`. 🤔

Reply:
Whoops, changed back to 2.3 for backwards compatibility